prow/plugins/golint/golint_test.go

```diff
@@ -55,6 +55,46 @@ func Qux() error {
 `),
 }
 
+var pullFilesForLintNames1 = map[string][]byte{
+	"qux.go": []byte(`// Package bar comment
+package bar
+
+// Qux_1 comment
+func Qux_1() error {
+	return nil
+}
+`),
+}
+
+var pullFilesForLintNames2 = map[string][]byte{
+	"qux.go": []byte(`// Package bar comment
+package bar
+
+// QUX_FUNC comment
+func QUX_FUNC() error {
+	return nil
+}
+`),
+}
+
+var codeChangeForLintNames1 = "@@ -0,0 +1,7 @@\n+// Package bar comment\n+package bar\n+\n+// Qux_1 comment\n+func Qux_1_Func() error {\n+   return nil\n+}"
+
+var lintNames1Suggestion = "```suggestion\nfunc Qux1() error {\n```"
+
+var codeChangeForLintNames2 = "@@ -0,0 +1,7 @@\n+// Package bar comment\n+package bar\n+\n+// QUX_FUNC comment\n+func QUX_FUNC() error {\n+   return nil\n+}"
+
+var lintNames2Suggestion = "```suggestion\nfunc QuxFunc() error {\n```"
+
+var lintCodeChangeMap = map[string]map[string][]byte {
+	codeChangeForLintNames1: pullFilesForLintNames1,
+	codeChangeForLintNames2: pullFilesForLintNames2,
+}
+// Expected lint suggestions for lint errors
+var lintSuggestionMap = map[string]string {
+	codeChangeForLintNames1: lintNames1Suggestion,
+	codeChangeForLintNames2: lintNames2Suggestion,
+}
```

Comment:
If these variables do not need to be globally scoped please move them into the functions where they are used.